src/indexer/handleEvent.ts

```diff
@@ -614,6 +641,76 @@ async function handleEvent(
       break;
     }
 
+    // --- Direct Payout Strategy
+    case "PayoutContractCreated": {
+      indexer.subscribe(
+        event.args.payoutContractAddress,
+        (
+          await import(
+            "#abis/v2/DirectPayoutStrategyImplementation.json",
+            {
+              assert: { type: "json" },
+            }
+          )
+        ).default,
+        event.blockNumber
+      );
+      break;
+    }
+
+    case "ApplicationInReviewUpdated": {
```

Comment:
Is there any chance of factoring out common code between the `ApplicationInReviewUpdated` handler and the `ApplicationStatusesUpdated` handler?